test_conformance/compiler/test_build_helpers.cpp

```diff
@@ -116,6 +117,13 @@ __kernel void sample_test_D(__global float *src, __global int *dst)
 #endif
 )";
 
+const char *sample_name_size_test_kernel = R"(
```

Comment:
Consider declaring this as a local variable in the test function itself - saves scrolling to find it.  Or, if you choose to keep it as a global variable, consider declaring it "static".